src/main/plugin/iso19139.ca.HNAP/schematron/schematron-rules-multilingual.sch

```diff
@@ -68,6 +68,24 @@
     <xsl:value-of select="$v" />
   </xsl:function>
 
+  <xsl:function name="geonet:openLicenseList" as="xs:string">
+    <xsl:param name="thesaurusDir" as="xs:string"/>
+
+    <xsl:variable name="licenseSeparator" select="if ($lang = 'fre') then ' ou ' else ' or '"/>
+
+    <xsl:variable name="open-license-list" select="document(concat('file:///', replace(concat($thesaurusDir, '/external/thesauri/theme/GC_Open_Licenses.rdf'), '\\', '/')))"/>
+
+    <xsl:variable name="v">
+      <xsl:for-each select="$open-license-list//rdf:Description/ns2:prefLabel[@xml:lang=$altLanguage2char]">
```

Comment:
I would expect $altLanguage2char to be replaced by a parameter supplied to the function to indicate the language of the list of values being supplied. Otherwise it seems like you will always get the alternate languages as a list.